src/website/src/pages/ec/components/hero-banner/docs/usage.mdx

```diff
@@ -0,0 +1,54 @@
+---
+title: Usage
+order: 1
+---
+
+import { Paragraph } from '@ecl/website-components';
+
+<Paragraph size="lead">
+  A Hero banner displays a prominent message and related optional actions.
+  Banners are displayed on the homepage below the site header of EC sites. It
+  conveys an important message to your audience. Hero banners are a good way to
+  channel your visitors to complete a specific task (ex: register to an event,
+  apply for funding... ). The user can choose to ignore them or interact with
+  them at any time.
+</Paragraph>
+
+## Anatomy
+
+Banners should have a focused message and provide users with clear actions to take. They should be placed at the top of a layout or screen. When scrolling, banners typically move with content and scroll off the screen. Only one banner should be shown at a time.
+
+![hero banner](https://inno-ecl.s3.amazonaws.com/media/images/EC/Hero%20Banner/Hero%20Banner01.jpg)
+
+Hero banner images have an aspect ratio of 3:1, meaning the width will be always three times the height of the images;
+
+![hero banner 2](https://inno-ecl.s3.amazonaws.com/media/images/EC/Hero%20Banner/Hero%20Banner02.png)
+
+| Components            | Use       | Restrictions                                              |
+| --------------------- | --------- | --------------------------------------------------------- |
+| hero image            | optional  | choice of 2 background colours when no image              |
+| description           | mandatory | 100-150 characters maximum, including punctuation.        |
+| call to action button | mandatory | 3-word max – 25 characters maximum, including punctuation |
+
+## When to use
+
+- Exclusively on the campaign websites
+- To communicate messages which are important but don’t require user action.
+- To communicate a change, an error, or an important event.
+
+## When not to use
+
+- Do not use more than one banner at a time.
+- Do not use banners as ‘alerts supporting a feedback mechanism’.
+- Don’t include more than one call to action in a banner.
+
+## DO's
+
+- The supporting illustration is
+
+  - relevant
+  - high quality
+  - emotionally persuasive
+  - the minimum ratio is 3:1
+
+## DON'Ts
```

Comment:
- Use call to action to compete with a bright, colourful image
- Use more than one banner in the homepage
- Use it in subpages